docs/en/operations/settings/settings.md

```diff
@@ -4008,6 +4008,35 @@ Default value: `0`.
 Use this setting only for backward compatibility if your use cases depend on old syntax.
 :::
 
+## session_timezone {#session_timezone}
+
+If specified, sets an implicit timezone (instead of server-default). All DateTime/DateTime64 values (and/or functions results) that have no explicit timezone specified are treated as having this timezone instead of default.
```

Comment:
Setting this to `''` (empty string), effectively resets implicit timezone to server timezone.